music/src/core/sequences.ts

```diff
@@ -288,3 +288,25 @@ export function assertIsQuantizedSequence(ns: INoteSequence) {
         `NoteSequence ${ns.id} is not quantized (missing quantizationInfo)`);
   }
 }
+
+export function isRelativeQuantizedSequence(ns: INoteSequence) {
```

Comment:
Can you just add a short description of relative vs absolute?

Reply:
Done